### PR TITLE
Added rohin's results

### PR DESCRIPTION
**Pull request by rohinmanvi**
- Final validation loss: 3.60
- Link: https://api.wandb.ai/links/rohinmanvi/0xenuz30
- Description: vanilla model with 8 layers + weight tying of input and output embeddings + hyperparameter optimization + torch.compile

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
